sol/private/sol_binary.bzl

```diff
@@ -72,11 +88,34 @@ def _gather_transitive_sources(attr):
             result.append(dep[SolSourcesInfo].transitive_sources)
     return result
 
+def _solc_meta(ctx):
```

Comment:
I think it would be better layering if the toolchain-provided SolInfo had the version, either as a string or a function that returns a string.

Reply:
Done.